pyenttec/test.py

```diff
@@ -1,15 +1,48 @@
 """Tests for offline DMX port."""
-from nose.tools import assert_equal, assert_raises
-from . import DMXConnectionOffline, DMXAddressError
+from nose.tools import assert_equal, assert_raises, raises
+from . import DMXConnectionOffline, DMXAddressError, DMXOverflowError
+
+def get_test_port(univ_size=24):
```

Comment:
All this function does is shadow the offline port constructor, and the kwarg default isn't used in any of the tests.  Since this is a very simple test fixture, please just call the constructor directly.